src/main/java/com/mcmoddev/orespawn/commands/AddOreCommand.java

```diff
@@ -51,77 +55,59 @@ public void execute(MinecraftServer server, ICommandSender sender, String[] args
         } else if (args.length != 2) {
             throw new CommandException(this.getUsage(sender));
         }
-
-        File file = new File(".", "orespawn" + File.separator + args[0] + ".json");
-        JsonParser parser = new JsonParser();
+        
+        String file = args[0];
         @SuppressWarnings("deprecation")
         IBlockState state = ((ItemBlock) stack.getItem()).getBlock().getStateFromMeta(stack.getItemDamage());
 
-        if (!file.exists()) {
-            throw new CommandException("That file doesn't exist" + (args[0].endsWith(".json") ? " (don't add .json)" : ""));
-        }
-
-        int dimension = OreSpawnAPI.DIMENSION_WILDCARD;
+        int dimension = OreSpawn.API.dimensionWildcard();
 
         try {
-            if (!args[1].equals(all)) {
+            if (!args[1].equals(ALL)) {
                 dimension = Integer.parseInt(args[1]);
             }
         } catch (NumberFormatException e) {
             throw new CommandException(args[1] + " isn't a valid dimension");
         }
 
         JsonObject ore = new JsonObject();
-        ore.addProperty("block", state.getBlock().getRegistryName().toString());
-        ore.addProperty("state", StateUtil.serializeState(state));
+        ore.addProperty(BLOCK, state.getBlock().getRegistryName().toString());
+        ore.addProperty(STATE2, StateUtil.serializeState(state));
         ore.addProperty("size", 25);
```

Comment:
We should probably extend addore so size et al can be specified as part of the command, or defaulted to as shown if not provided